test/e2e/storage/testsuites/testdriver.go

```diff
@@ -59,7 +59,7 @@ type TestVolume interface {
 type PreprovisionedVolumeTestDriver interface {
 	TestDriver
 	// CreateVolume creates a pre-provisioned volume of the desired volume type.
-	CreateVolume(volumeType testpatterns.TestVolType) TestVolume
+	CreateVolume(config *TestConfig, volumeType testpatterns.TestVolType) TestVolume
```

Comment:
same as with CreateDriver, I think it would be clearer to return a TestConfig if CreateVolume may modify the config

Reply:
yes

Reply:
Not done yet.